sections/03_categorize.md

```diff
@@ -282,36 +281,42 @@ learning in the domain. Though not a methodological consideration, efforts are
 underway to recognize those who promote an ecosystem of rigorous sharing and
 analysis [@doi:10.1038/ng.3830].
 
-We expect both improved methods and an improved culture of sharing to
-play an important role if deep learning is going to transform how we analyze
-data to categorize states of human health. We don't expect that deep learning
-methods will replace expert review. We expect them to complement expert review
-by allowing more efficient use of the costly practice of manual annotation.
+We expect both improved methods and an improved culture of sharing to play an
```

Comment:
Can you also adjust this paragraph to match the revisions we made before - I think these lines are the most up to date:
https://github.com/greenelab/deep-review/blame/master/sections/03_categorize.md#L292